From 0c2b9a4a2df1d42ab8ebfaf340bbfeeed1268c48 Mon Sep 17 00:00:00 2001
From: Will Newton <will.newton@gmail.com>
Date: Thu, 17 Sep 2009 16:45:11 +0000
Subject: python: Don't delete .py files unless asked.

If we have set BR2_PACKAGE_PYTHON_PY_ONLY or BR2_PACKAGE_PYTHON_PY_PYC
we want .py files to be left on the target, whatever the value of
BR2_PACKAGE_PYTHON_DEV.

Signed-off-by: Will Newton <will.newton@gmail.com>
---
diff --git a/package/python/python.mk b/package/python/python.mk
index cd1ccec..95586ad 100644
--- a/package/python/python.mk
+++ b/package/python/python.mk
@@ -155,7 +155,6 @@ endif
 ifneq ($(BR2_PACKAGE_PYTHON_DEV),y)
 	rm -rf $(TARGET_DIR)/usr/include/python$(PYTHON_VERSION_SHORT)
 	rm -rf $(TARGET_DIR)/usr/lib/python$(PYTHON_VERSION_SHORT)/config
-	find $(TARGET_DIR)/usr/lib/ -name '*.py' -exec rm {} \;
 endif
 ifneq ($(BR2_PACKAGE_PYTHON_BSDDB),y)
 	rm -rf $(TARGET_DIR)/usr/lib/python$(PYTHON_VERSION_MAJOR)/bsddb
--
cgit v0.8.2.1
